config/routes.rb

```diff
@@ -4,7 +4,9 @@
 
   resources :landlords
   resources :tenants
+  resources :static
```

Comment:
These are not really resources per say. Resources are things that have crud actions: create update destroy (index show).

The way we would probably display these in our routes files would be something like this:
`get 'home', to: 'static#home'`
` get 'about', to: 'static#about'`